homeassistant/components/zwave/util.py

```diff
@@ -1,54 +1,71 @@
 """Zwave util methods."""
 import logging
 
+from . import const
+
 _LOGGER = logging.getLogger(__name__)
 
 
-def value_handler(value, method=None, class_id=None, index=None,
-                  label=None, data=None, member=None, instance=None,
-                  **kwargs):
-    """Get the values for a given command_class with arguments.
+def check_node_schema(node, schema):
+    """Check if node matches the passed node schema."""
+    if (const.DISC_NODE_ID in schema and
```

Comment:
Would it be possible to make the schemas voluptuous schemas and just validate here instead?